doc/source/user_guide/copy_on_write.rst

```diff
@@ -116,10 +116,32 @@ The following code snippet updated both ``df`` and ``subset`` without CoW:
 
 This is not possible anymore with CoW, since the CoW rules explicitly forbid this.
 This includes updating a single column as a :class:`Series` and relying on the change
-propagating back to the parent :class:`DataFrame`.
-This statement can be rewritten into a single statement with ``loc`` or ``iloc`` if
-this behavior is necessary. :meth:`DataFrame.where` is another suitable alternative
-for this case.
+propagating back to the parent :class:`DataFrame`. To modify a DataFrame value in a given
+column and row, the code must be rewritten as a single assignment to ``loc`` or ``iloc``.
+When the column is given by name (``loc``) and the row by position (``iloc``), you either
+need to convert the column name to its position using :meth:`Index.get_loc` or you need
+to convert the row position to its index. Both variants as shown in the following snippet:
+
+.. code-block:: ipython
```

Comment:
We only use `code-block` when demonstrating old behavior that is no longer the case. Can you instead use `.. ipython:: python` (see other sections of this file for examples).